lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/src/main/java/io/lighty/modules/gnmi/simulatordevice/impl/SimulatedGnmiDeviceBuilder.java

```diff
@@ -101,10 +44,29 @@ public SimulatedGnmiDeviceBuilder setSupportedEncodings(final EnumSet<Gnmi.Encod
         return this;
     }
```

Comment:
move also `bossGroup`, `workerGroup`, `encondigs` and `gson` to configuration DTO,
then this builder will consist of `from()` and `buidl()` method
This adds possibility to programmatically fully-configure the device with just config DTO.